crypto/fipsmodule/ec/ec_nistp.c

```diff
@@ -17,9 +17,9 @@
 // |----------------------------|
```

Comment:
Can remove this tracking table now?
Can be follow-up.

Reply:
will do, thanks!